crates/client/Cargo.toml

```diff
@@ -19,7 +19,7 @@ test-support = ["clock/test-support", "collections/test-support", "gpui/test-sup
 anyhow.workspace = true
 async-recursion = "0.3"
 async-tungstenite = { version = "0.16", features = ["async-std", "async-native-tls"] }
-async-native-tls = { version = "0.5.0", features = ["vendored"] }
```

Comment:
Does this impact the macOS builds at all? Is this something we only want to do on Linux?

Reply:
WIP change :)